tests/integration/pilot/common/routing.go

```diff
@@ -3497,26 +3497,27 @@ spec:
 
 	for _, client := range flatten(t.Apps.VM, t.Apps.A, t.Apps.Tproxy) {
 		v4, v6 := getSupportedIPFamilies(t, client)
-		var expectedIPv4 []string
-		var expectedIPv6 []string
-		if v4 && v6 {
-			expectedIPv4 = []string{"1.2.3.4", "1.2.3.5"}
-			expectedIPv6 = []string{"1234:1234:1234::1234:1234:1234", "1235:1235:1235::1235:1235:1235"}
-		} else if v4 {
-			expectedIPv4 = []string{"1.2.3.4", "1.2.3.5"}
-			expectedIPv6 = []string{"1234:1234:1234::1234:1234:1234"}
```

Comment:
This seems not right, i need to filterout GetAddressForProxy() respecting the node supported ip mode